drivers/node/vsphere/vsphere.go

```diff
@@ -206,9 +206,10 @@ func (v *vsphere) RebootNode(n node.Node, options node.RebootNodeOpts) error {
 	if _, ok := vmMap[n.Name]; !ok {
 		return fmt.Errorf("could not fetch VM for node: %s", n.Name)
 	}
-
 	vm := vmMap[n.Name]
-
+	//Reestblish connection to avoid session timeout.
+	v.connect()
+	vm = vmMap[n.Name]
```

Comment:
remove the redunant assignment

Reply:
Thanks for the review, addressed review comments